YapDatabase/YapDatabase.m

```diff
@@ -72,6 +72,11 @@
 NSString *const YapDatabaseExtensionDependenciesKey  = @"extensionDependencies";
 NSString *const YapDatabaseNotificationKey           = @"notification";
 
+NSString *const YapDBUnexpectedNilObjectNotification = @"YapDatabaseModifiedExternallyNotification";
```

Comment:
Use the same string value as the identifier name.

Also let's use `YapDatabase` instead of `YapDB` prefix to be more consistent with other keys' naming

Reply:
renamed.

Reply:
Also renamed the notification string to align with the variable name.